homeassistant/const.py

```diff
@@ -292,6 +292,11 @@
 # Temperature attribute
 ATTR_TEMPERATURE = 'temperature'
 
+# for onvif platform's PTZ
+ATTR_DIR_H = 'dir_h'
+ATTR_DIR_V = 'dir_v'
+ATTR_ZOOM = 'zoom'
```

Comment:
Those constants are only used by the `ONVIF` camera platform. Move them to `onvif.py`.

Reply:
As I'm thinking about it... Maybe I'll rename DIR_H to PAN, DIR_V to TILT?
English is not my native language and when I was writing the code it seemed ok, but now as I'm thinking about it... Pan and tilt sound better. What do you think?